sysfs/class_drm_amdgpu.go

```diff
@@ -47,6 +52,38 @@ type ClassDRMCardAMDGPUStats struct {
 	MemoryVRAMVendor              string // The VRAM vendor name.
 	PowerDPMForcePerformanceLevel string // The current power performance level.
 	UniqueID                      string // The unique ID of the GPU that will persist from machine to machine.
+	HWmonChip                     string // The hwmon chip.
+}
+
+func cleanChipName(name string) string {
```

Comment:
Why do we need to sanitize the name?

Reply:
It's something that the hwmon does with the chip names as well, so the format is similar. 

See: https://github.com/prometheus/node_exporter/blob/e6a9cfbdcdaa21bf9676c6cd37bef8160227f423/collector/hwmon_linux.go#L74

Reply:
Yes but in that case its used to construct the metric name, which has more limitations. Here its being used as label value, where I don't think we need this

Reply:
Do you think it will still be the same and relatable by the chip from the hwmon? It seems like a good idea to align the result to what the hwmon does with its chip names

Reply:
I guess in the node-exporter we should also just use the raw chip name and put it in a label instead of the metric name. But I think either way, here, this being used as label value, it should not get santized. But want tohave @SuperQ opinion on this as well.

Reply:
A clarification - I don't care if it is sanitized or not; the only reason I am doing a sanitization here is to align the label `chip` coming from the hwmon metrics (which is also sanitized as per the permalink I provided above) to a new label I've added to the `node_drm_card_info` metric. 

I want to match, aka join a metric like `node_hwmon_fan_rpm` with `node_drm_card_info`  and have a connection between the two collectors. RN, there is no way to match the metrics coming from those collectors, although both provide metrics for GPUs. 

Sanitizing the label before the match makes sense because this is what's being done for the `chip` label in the hwmon collector. 

If you think we shouldn't sanitize the hwmon chip label in the node_exporter, I might provide a PR for that. As of now, I am aligning the behaviour with the existing solution.

Reply:
Yeah makes sense, let's see what @SuperQ is thinking about this. I'm fine either way but moving chip to a label in the hwmon collector makes most sense to me.

Reply:
@SuperQ Please take a look